### PR TITLE
[zh] Resync kubeadm files (1)

### PR DESCRIPTION
**Pull request by tengqm**
related: #27670

### CODE REVIEW

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 1]
```diff
@@ -196,142 +202,121 @@ option. Your cluster requirements may need a different configuration.
...
-    为了安全起见，将在两个小时内删除上传的证书； 如有必要，您可以使用 kubeadm 初始化上传证书阶段，之后重新加载证书。
-
-    然后，您可以通过在根目录上运行以下命令来加入任意数量的工作节点：
-    kubeadm join 192.168.0.200:6443 --token 9vr73a.a8uxyaju799qwdjv --discovery-token-ca-cert-hash sha256:7c2e69131a36ae2a042a339b33381c6d0d43887e2de83720eff5359e26aec866
-    ```
-
-    - 将此输出复制到文本文件。 稍后您将需要它来将控制平面节点和辅助节点加入集群。
-    - 当 `--upload-certs` 与 `kubeadm init` 一起使用时，主控制平面的证书被加密并上传到 `kubeadm-certs` 密钥中。
+    -->
+    - 将此输出复制到文本文件。 稍后你将需要它来将控制平面节点和辅助节点加入集群。
```
Comment (SataQiu) [inline content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md:264]:
```suggestion
    - 将此输出复制到文本文件。 稍后你将需要它来将控制平面节点和工作节点加入集群。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 2]
```diff
@@ -432,93 +416,86 @@ in the kubeadm config file.
...
+      - `LOAD_BALANCER_DNS`
+      - `LOAD_BALANCER_PORT`
+      - `ETCD_0_IP`
+      - `ETCD_1_IP`
+      - `ETCD_2_IP`
 
+<!--
 The following steps are similar to the stacked etcd setup:
+-->
+以下的步骤与设置内置 ETC的 集群是相似的：
```
Comment (SataQiu) [inline content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md:481]:
```suggestion
以下的步骤与设置内置 ETCD 的集群是相似的：
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 3]
```diff
@@ -529,34 +506,30 @@ The steps are the same as for the stacked etcd setup:
...
 as the output from the `kubeadm init` command:
-
-```sh
-sudo kubeadm join 192.168.0.200:6443 --token 9vr73a.a8uxyaju799qwdjv --discovery-token-ca-cert-hash sha256:7c2e69131a36ae2a042a339b33381c6d0d43887e2de83720eff5359e26aec866
-```
-
 -->
-您可以使用之前存储的命令将工作节点加入集群中
-作为 `kubeadm init` 命令的输出：
+你可以使用之前存储的、`kubeadm init` 命令的输出将工作节点加入集群中：
```
Comment (SataQiu) [inline content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md:532]:
```suggestion
你可以使用之前存储的 `kubeadm init` 命令的输出将工作节点加入集群中：
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md [hunk 4]
```diff
@@ -33,10 +31,11 @@ manager instead, but you need to configure it manually.
...
 other configuration aspects need to be set on a per-kubelet basis to accommodate the different
 characteristics of a given machine (such as OS, storage, and networking). You can manage the configuration
-of your kubelets manually, but kubeadm now provides a `KubeletConfiguration` API type for [managing your
-kubelet configurations centrally](#configure-kubelets-using-kubeadm).
+of your kubelets manually, but kubeadm now provides a `KubeletConfiguration` API type for
+[managing your kubelet configurations centrally](#configure-kubelets-using-kubeadm).
 -->
-kubeadm CLI 工具的生命周期与 [kubelet](/zh/docs/reference/command-line-tools-reference/kubelet)解耦，它是一个守护程序，在 Kubernetes 集群中的每个节点上运行。
+kubeadm CLI 工具的生命周期与 [kubelet](/zh/docs/reference/command-line-tools-reference/kubelet)
+解耦；它是一个守护程序，在 Kubernetes 集群中的每个节点上运行。
```
Comment (SataQiu) [inline content/zh/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md:38]:
```suggestion
解耦；kubelet 是一个守护程序，在 Kubernetes 集群中的每个节点上运行。
```

**Review by SataQiu — COMMENTED** [review]
Thanks @tengqm ！
LGTM, other than some nits.

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 5]
```diff
@@ -196,142 +202,121 @@ option. Your cluster requirements may need a different configuration.
...
+    Some CNI network plugins like Calico require a CIDR such as `192.168.0.0/16` and
+    some like Weave do not. See the [CNI network documentation](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network).
+    To add a pod CIDR pass the flag `--pod-network-cidr`, or if you are using a kubeadm configuration file
+    set the `podSubnet` field under the `networking` object of `ClusterConfiguration`.
+    -->
+    一些 CNI 网络插件如 Calico 需要 CIDR 例如 `192.168.0.0/16` 和一些像 Weave 没有。参考
+    [CNI 网络文档](/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network)。
+    通过传递 `--pod-network-cidr` 标志添加 pod CIDR，或者你可以使用 kubeadm
+    配置文件，在 `ClusterConfiguration` 的 `networking` 对象下设置 `podSubnet` 字段。
+    {{< /note >}}
```
Comment (SataQiu) [inline content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md:238]: There are some page layout problems here


image

Comment (tengqm) [reply content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md:238]: done

File: content/zh/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md (hunk 4 above)
Comment (tengqm) [reply content/zh/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md:38]: done

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md (hunk 3 above)
Comment (tengqm) [reply content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md]: done

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md (hunk 2 above)
Comment (tengqm) [reply content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md]: done

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md]: fixed

File: content/zh/docs/reference/config-api/kubelet-config.v1beta1.md [hunk 6]
```diff
@@ -0,0 +1,1604 @@
+---
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/reference/config-api/kubelet-config.v1beta1.md:1]: Have to wait for the _index.md file to come in. Not scope of this PR.

**Review by SataQiu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**